CONTRIBUTORS.md

```diff
@@ -1396,16 +1401,6 @@ Noida
 - Bio: Front-end developer
 - GitHub: [Lisa Nguyen](https://github.com/LisaNguyen)
 
-### Name: [Peter Squicciarini](https://github.com/stripedpajamas)
```

Comment:
Why are you deleting these names????

Reply:
Sorry, it´s my first time

Reply:
Do you think you could create a commit adding them back?

Reply:
Sure, let me figure it out

Reply:
```
#### Name: [Türker Yıldırım](https://github.com/turkerdotpy)		
 - Place: Tekirdağ, Turkey		
 - Bio: Literally gamer, geek and viking.		
 - GitHub: [turkerdotpy](https://github.com/turkerdotpy)		
```
Here they are. Copy the code into the readme wherever you see fit(just not in between contributions).